handlers/main.yml

```diff
@@ -1,6 +1,12 @@
 # Copyright 2015 VMware, Inc.  All rights reserved.
 # SPDX-License-Identifier: MIT OR GPL-3.0-only
 ---
+- name: reset failed kubelet
+  become: true
+  command: systemctl reset-failed kubelet
+  notify:
+    - restart kubelet
```

Comment:
Handlers notifying other handlers, is that a documented feature? I believe the conventional way to run multiple handler tasks is to `notify` both of them (`notify` takes a list of handlers, and handlers are run in the order they are defined and not in the order they are notified). Or using the `listen` directive.

Reply:
Hi @naveens,  I am just doing exactly what was done before me for kubelet restart in other roles. Please, look at handlers for the kubelet role or kubernetes-minion role for example.